lib/src/tasks/test/cli.dart

```diff
@@ -44,6 +44,11 @@ class TestCli extends TaskCli {
         negatable: true,
         defaultsTo: defaultPubServe,
         help: 'Serves browser tests using a Pub server.')
+    ..addFlag('pause-after-load',
+        abbr: 'w',
+        negatable: true,
```

Comment:
The `--pause-after-load` flag from the `test` package has no abbreviation and is not negatable, mind if we stay consistent with that? (https://github.com/dart-lang/test/blob/17cd53ebf069b27e25d609aa431f6abe6bf0f68f/lib/src/runner/configuration/args.dart#L83-L87)